src/applications/personalization/dashboard/reducers/index.js

```diff
@@ -14,11 +14,11 @@ import { combineReducers } from 'redux';
 export default {
   ...claimsAppeals,
   preferences,
-  appointments,
   ...profile,
   ...ratedDisabilities,
   hcaEnrollmentStatus,
   health: combineReducers({
+    appointmentsReducer,
```

Comment:
Not too fond of the naming, but I do prefer this over `appointments.appointments`

Reply:
I agree that we should rethink the naming, but I am also not a fan of `data`! I guess it is slightly better.. will rename to `data`